src/router.jsx

```diff
@@ -1,11 +1,13 @@
-import { createBrowserRouter, Navigate, Outlet } from "react-router-dom"
+import { createBrowserRouter, Navigate, Outlet } from "react-router-dom";
+import WelcomePage from "./components/Welcome-Page";
```

Comment:
Please move the `WelcomePage `into the `src/pages` folder.  If the `pages `folder doesn’t exist yet you can create it within `src`

Reply:
done